test_sum_turms.py

```diff
@@ -0,0 +1,14 @@
+'''Test for sum_of_nth_terms.py.'''
+test_list = [
+    [0, '0.00'],
+    [1, '1.00'],
+    [2, '1.25'],
+    [3, '1.39'],
+]
+
+
+def test_series_sum():
+    '''Test that series_sum returns the correct answer from the table.'''
+    from sum_of_nth_terms.py import series_sum
```

Comment:
This is not a valid import statement. Missed this the first time around, but there's no `sum_of_nth_terms.py` module. The module name is `sum_of_nth_terms`. Your tests fail.

Even with the name change, your tests still fail because you're not actually using the parametrized table like you're probably attempting to.